tests/testees/loop.rs

```diff
@@ -0,0 +1,13 @@
+#[inline(never)]
+#[no_mangle]
+fn breakpoint(counter: u8) -> () {
+    println!("- {} -", counter);
+}
+
+fn main() {
+    let breaks = 8;
+    for i in 0..breaks {
+        breakpoint(i);
+    }
+    println!("Called `breakpoint` {} times.", breaks - 1);
```

Comment:
```suggestion
    println!("Called `breakpoint` {} times.", breaks);
```

The range starts from 0.